packages/react-core/src/components/TimePicker/examples/TimePicker.md

```diff
@@ -0,0 +1,55 @@
+---
+id: Time picker
+section: components
+cssPrefix: pf-c-time-picker
+propComponents: ['TimePicker']
+beta: true
+---
+
+import { TimePicker } from '@patternfly/react-core';
+
+## Examples
+### Basic 12 hour
+```js
+import React from 'react';
+import { TimePicker } from '@patternfly/react-datetime';
+
+SimpleTimePicker = () => {
+  const onChange = (time) => {
+    console.log(time);
+  };
+  
+  return (
+    <React.Fragment>
+        <TimePicker defaultTime="3:35pm" onChange={onChange}/>
```

Comment:
I'm seeing `3:35am` instead of `3:35pm`
![image](https://user-images.githubusercontent.com/47335686/101376071-20aa5280-387e-11eb-93c7-6aaa56dd5c62.png)